src/resource.coffee

```diff
@@ -114,12 +116,19 @@ class Resource extends Model
         ids ?= if Array.isArray resourceLink
           resourceLink
 
+        href = null if type is 'users'
```

Comment:
Why're users special here? I'd like to keep any knowledge about the data itself out of the library.